frontend/src/components/user-management/SelectFormInputClassroom.tsx

```diff
@@ -0,0 +1,73 @@
+import {
```

Comment:
since this looks very similar to `SelectFormInput`, can you look into creating a generalized component for these two? you can set the props to be a union (i.e. `SelectFormInputProps | SelectFormInputClassroomProps`) and `resetError` as optional